### DISCUSSION TITLE
Configuring Seq HTTPS on Docker with haproxy

### DISCUSSION THREAD

**Discussion by nikimarchin** [datalust/seq-tickets#2317] [Help]

**Comment by nblumhardt** [comment]
Hi Nikolay, thanks for your message.

```
image: datalust/seq:5.1
```

This is a very old Seq image; 2024.3 is the current release.

```
    #ports:
    #  - "5341:80"
    #  - "45341:443"
```

If you want to access Seq on 80 and 443, try `expose`, which [should make those ports available to services on the same network without binding ports on the host machine](https://stackoverflow.com/questions/40801772/what-is-the-difference-between-ports-and-expose-in-docker-compose):

```
    expose:
      - "80"
      - "443"
```

I'm not especially familiar with Docker Compose and the differences between its major versions so you may need to dig around in the Compose docs for this.

To get 443 to be served on the Seq side, you'll need to mount that PEM file into the `/data/Certificates` directory in the container.

Hope this helps,
Nick
